go.mod

```diff
@@ -314,7 +314,7 @@ require (
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475 // indirect
 	github.com/rivo/uniseg v0.4.4 // indirect
 	github.com/rs/zerolog v1.29.1 // indirect
-	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/russross/blackfriday/v2 v2.1.0
```

Comment:
can you move this to the main require block instead of the `indirect` require block?

Reply:
Nice catch! I didn't notice that `go mod` left it there.

Reply:
I'm thinking it might be a bug in Go `1.21`